### PR TITLE
Mobile regex

### PR DESCRIPTION
**Pull request by RiaanWest**
[Review on Reviewable](https://reviewable.io/reviews/kagiso-future-media/django_auth/38)

### CODE REVIEW

File: kagiso_auth/forms.py [hunk 1]
```diff
@@ -56,11 +56,16 @@ class SignUpForm(forms.Form):
     )
     password = forms.CharField(widget=forms.PasswordInput(), min_length=8)
     confirm_password = forms.CharField(widget=forms.PasswordInput())
-    mobile = forms.CharField(  # TODO: RegexField?
+    mobile = forms.RegexField(
+        regex=r'^\d{10}$',
```
Comment (RiaanWest) [inline kagiso_auth/forms.py]: Reason for two regex's: HTML 5 client-side validation, and server-side Django validation

**Comment by RiaanWest** [comment]
Ready @lee-kagiso

**Comment by RiaanWest** [comment]
Reason for two regex's and two error messages: HTML 5 client-side validation, and server-side Django validation

**Comment by lee-kagiso** [comment]
you haven't removed the actual 3p lib yet from the repo

**Comment by RiaanWest** [comment]
3P Lib?

**Comment by lee-kagiso** [comment]
input-whatever.js

**Comment by RiaanWest** [comment]
That sits in CMS: https://github.com/Kagiso-Future-Media/cms/pull/637

**Comment by lee-kagiso** [comment]
ok can you remove it please too

**Comment by RiaanWest** [comment]
That's a link to a PR that also needs your attention :)

**Pull request merged**